src/mlpack/methods/ann/augmented/tasks/add_impl.hpp

```diff
@@ -39,14 +39,18 @@ AddTask::AddTask(const size_t bitLen) : bitLen(bitLen) {
 
 void AddTask::Generate(arma::field<arma::mat>& input,
                        arma::field<arma::mat>& labels,
-                       const size_t batchSize)
+                       const size_t batchSize,
+                       bool fixedLength = false)
```

Comment:
Default parameters can be added only in ``.hpp`` files and not in ``.cpp`` or ``impl.hpp`` files.